tests/integ/common/storage.go

```diff
@@ -7,15 +7,21 @@ package common
 
 import (
 	"fmt"
+	"os"
 
 	"github.com/facebookincubator/contest/pkg/storage"
 	"github.com/facebookincubator/contest/plugins/storage/rdbms"
 )
 
-const DbURI = "contest:contest@tcp(mysql:3306)/contest_integ?parseTime=true"
+const DbURI = "contest:contest@tcp(localhost:3306)/contest_integ?parseTime=true"
+const DbURICI = "contest:contest@tcp(mysql:3306)/contest_integ?parseTime=true"
 
 func NewStorage(opts ...rdbms.Opt) (storage.Storage, error) {
-	return rdbms.New(DbURI, opts...)
+	if os.Getenv("CI") != "" {
+		return rdbms.New(DbURICI, opts...)
+	} else {
+		return rdbms.New(DbURI, opts...)
```

Comment:
Thanks for this, I replaced this string a hundred times already and every. single. time. I end up committing it by accident. Would you mind doing the same in `dblocker_test.go` as well?

Reply:
done